src/main.rs

```diff
@@ -636,7 +614,74 @@ fn method_to_filter(method: Method) -> MethodFilter {
         m => panic!("Unsupported HTTP method: {m:?}"),
     }
 }
+fn server_routes() -> Router {
```

Comment:
```suggestion
fn server_server_routes() -> Router {
```

Just so it's more clear that these are federation/S2S routes only.